src/sbvr-api/translations.ts

```diff
@@ -208,24 +208,36 @@ export const translateAbstractSqlModel = (
 		if (translationDefinition) {
 			const { $toResource, ...definition } = translationDefinition;
 			const hasToResource = typeof $toResource === 'string';
+			let aliasedToResource;
 			if (hasToResource) {
-				if ($toResource.endsWith(toVersionSuffix)) {
-					// Ideally we want to rename to the unaliased resource of the next version
-					// so when the alias matches the next version we can just strip it
-					resourceRenames[key] = $toResource.slice(0, -toVersionSuffix.length);
+				if ($toResource.includes('$')) {
+					// TODO-MAJOR: Change this to an error
+					console.warn(
+						`'$toResource' should be the unaliased name of the resource in the subsequent model and not be targeting a specific model, got '${$toResource}'`,
+						$toResource,
+					);
+					if ($toResource.endsWith(toVersionSuffix)) {
+						// Ideally we want to rename to the unaliased resource of the next version
+						// so when the alias matches the next version we can just strip it
+						resourceRenames[key] = $toResource.slice(
+							0,
+							-toVersionSuffix.length,
+						);
+					}
```

Comment:
Afaict, if we also add v then oB's master would not break
```suggestion
					} else {
						resourceRenames[key] = $toResource;
					}
```